website/docs/reference/commands/deps.md

```diff
@@ -60,26 +60,34 @@ Update your versions in packages.yml, then run dbt deps
 
 <VersionBlock firstVersion="1.7">
 
-dbt generates the `package-lock.yml` file in the _project_root_ where `packages.yml` is recorded, which contains all the resolved packages, the first time you run `dbt deps`. Each subsequent run records the packages installed in this file. If the subsequent `dbt deps` runs contain no updated packages in `dependencies.yml` or `packages.yml`, dbt-core installs from `package-lock.yml`. 
+## Predictable package installs
 
-When you update the package spec and run `dbt deps` again, the `package-lock.yml` and `packages.yml` files update accordingly. 
+Starting in dbt Core v1.7, dbt generates an additional `package-lock.yml` file in the root of your project. This contains the full set of resolved packages, based on the `packages` configuration (in `dependencies.yml` or `packages.yml`). Each subsequent invocation of `dbt deps` will install from the _locked_ set of packages specified in this file. By storing the full set of required packages (with pinned versions) in version-controlled code, it ensures predictable installs in production and consistency across all developers and environments.
```

Comment:
Suggest removing "additional" so it doesn't read like there will be 2 package-lock.yml files in the project
```suggestion
Starting in dbt Core v1.7, dbt generates a `package-lock.yml` file in the root of your project. This contains the complete set of resolved packages based on the `packages` configuration in `dependencies.yml` or `packages.yml`. Each subsequent invocation of `dbt deps` will install from the _locked_ set of packages specified in this file. Storing the complete set of required packages (with pinned versions) in version-controlled code ensures predictable installs in production and consistency across all developers and environments.
```